omod/src/main/webapp/configureDhis2.jsp

```diff
@@ -1,7 +1,17 @@
 <%@ include file="/WEB-INF/template/include.jsp"%>
 <%@ include file="/WEB-INF/template/header.jsp"%>
-
 <%@ include file="template/localHeader.jsp"%>

```

Comment:
same as above - this change should not come up in pull requestssame as above - this change should not come up in pull requests